src/plugins/dashboard/server/content_management/v3/cm_services.ts

```diff
@@ -0,0 +1,540 @@
+    schema.string({ meta: { description: 'The saved object id for by reference panels' } })
+  ),
+  type: schema.string({ meta: { description: 'The embeddable type' } }),
+  panelRefName: schema.maybe(schema.string()),
+  gridData: gridDataSchema,
+  panelIndex: schema.string({
+    meta: { description: 'The unique ID of the panel.' },
+    defaultValue: schema.siblingRef('gridData.i'),
+  }),
+  title: schema.maybe(schema.string({ meta: { description: 'The title of the panel' } })),
+  version: schema.maybe(
+    schema.string({
+      meta: {
+        description:
+          "The version was used to store Kibana version information from versions 7.3.0 -> 8.11.0. As of version 8.11.0, the versioning information is now per-embeddable-type and is stored on the embeddable's input. (panelConfig in this type).",
+        deprecated: true,
+      },
+    })
+  ),
+});
+
+export const optionsSchema = schema.object({
+  hidePanelTitles: schema.boolean({
+    defaultValue: DEFAULT_DASHBOARD_OPTIONS.hidePanelTitles,
+    meta: { description: 'Hide the panel titles in the dashboard.' },
+  }),
+  useMargins: schema.boolean({
+    defaultValue: DEFAULT_DASHBOARD_OPTIONS.useMargins,
+    meta: { description: 'Show margins between panels in the dashboard layout.' },
+  }),
+  syncColors: schema.boolean({
+    defaultValue: DEFAULT_DASHBOARD_OPTIONS.syncColors,
+    meta: { description: 'Synchronize colors between related panels in the dashboard.' },
+  }),
+  syncTooltips: schema.boolean({
+    defaultValue: DEFAULT_DASHBOARD_OPTIONS.syncTooltips,
+    meta: { description: 'Synchronize tooltips between related panels in the dashboard.' },
+  }),
+  syncCursor: schema.boolean({
+    defaultValue: DEFAULT_DASHBOARD_OPTIONS.syncCursor,
+    meta: { description: 'Synchronize cursor position between related panels in the dashboard.' },
+  }),
+});
+
+// These are the attributes that are returned in search results
+export const searchResultsAttributesSchema = schema.object({
+  title: schema.string({ meta: { description: 'A human-readable title for the dashboard' } }),
+  description: schema.string({ defaultValue: '', meta: { description: 'A short description.' } }),
+  timeRestore: schema.boolean({
+    defaultValue: false,
+    meta: { description: 'Whether to restore time upon viewing this dashboard' },
+  }),
+});
+
+export const dashboardAttributesSchema = searchResultsAttributesSchema.extends({
+  // Search
+  kibanaSavedObjectMeta: schema.object(
+    {
+      searchSource: schema.maybe(searchSourceSchema),
+    },
+    {
+      meta: {
+        description: 'A container for various metadata',
+      },
+      defaultValue: {},
+    }
+  ),
+  // Time
+  timeFrom: schema.maybe(
+    schema.string({ meta: { description: 'An ISO string indicating when to restore time from' } })
+  ),
+  timeTo: schema.maybe(
+    schema.string({ meta: { description: 'An ISO string indicating when to restore time from' } })
+  ),
+  refreshInterval: schema.maybe(
```

Comment:
Should refresh interval schema be imported from es-query package

Reply:
To avoid leaking the `@kbn/config-schema` server package in the browser, we would need to define a separate `es-query-server` package and export the schema type from there. Perhaps that is a future goal as we expose more public APIs from our analytics plugins. But I think the duplication of this type is fine for Dashboards so we don't couple too strongly to a schema that might incur future changes from other plugins.